.vscode/settings.json

```diff
@@ -7,6 +7,9 @@
     "editor.formatOnSave": true,
     "editor.insertSpaces": true,
     "editor.tabSize": 4,
+    "files.exclude": {
```

Comment:
Should you just ignore the entire testFolder instead of any directory that includes "hello-world"?

Reply:
I want the README to display otherwise I feel like people might be confused why the folder shows as empty